src/test/java/com/microsoft/azure/spring/data/cosmosdb/repository/integration/RoleRepositoryCollectionIT.java

```diff
@@ -48,13 +61,25 @@ public void testBasicQuery() {
     }
 
     @Test
-    public void testSaveAll() {
-        this.repository.saveAll(Collections.singleton(DOMAIN));
+    public void testSaveAndFindById() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/Microsoft/spring-data-cosmosdb/pullRequest?prid=4925697)